error.go

```diff
@@ -24,3 +27,14 @@ func IsNotFound(err error) bool {
 
 	return false
 }
+
+// IsAlreadyExists returns true if the error message and error code that indicates that entity already exists
+func IsAlreadyExists(err error) bool {
+	if e, ok := err.(Error); ok {
+		if strings.Contains(e.Message, "already exists") && (e.Status == 409 || e.Status == 403) {
```

Comment:
Which APIs return 403 when a resource exists? It might be there is some but that doesn't sound like something that should be returned.

Reply:
**POST** `/v1/project/<pname>/service/<sname>/user` retrieves 403, perhaps some other endpoints that I do not remember.

Reply:
Yes looks like creating new user, database or connection pool currently indeed returns 403 in this case. That seems like nonsensical code. I'll check if it is something that should be changed.

Reply:
There's open pull request for changing the 403 errors into 409. That is still pending for some clients to be updated not to rely on 403 but will hopefully get merged in next couple of days.

Reply:
Great, thanks! I have updated code only to use 409 error code.